### PR TITLE
keycloak_quarkus: systemd restart behavior

### LINKED ISSUE #144: keycloak_quarkus: Introduce systemd service (restart) behavior
<title, PR incoming>

### PR DESCRIPTION
**Pull request by hwo-wd**
Fix #144

### CODE REVIEW

File: roles/keycloak_quarkus/README.md [hunk 1]
```diff
@@ -79,15 +82,15 @@ Role Defaults
...
+|`keycloak_quarkus_ispn_trust_store_password` | Password for infinispan certificate keystore | `changeit` |
 
 
 * Install options
 
 | Variable | Description | Default |
 |:---------|:------------|:---------|
 |`keycloak_quarkus_offline_install` | Perform an offline install | `False`|
-|`keycloak_quarkus_download_url`| Download URL for keycloak | `https://github.com/keycloak/keycloak/releases/download/<version>/<archive>`| 
+|`keycloak_quarkus_download_url`| Download URL for keycloak | `https://github.com/keycloak/keycloak/releases/download/<version>/<archive>`|
```
Comment (guidograzioli) [inline roles/keycloak_quarkus/README.md:93]: Dup few lines below (please keep that with the jinja vars)

File: roles/keycloak_quarkus/README.md [hunk 2]
```diff
@@ -108,7 +111,6 @@ Role Defaults
 |`keycloak_auth_client` | Authentication client for configuration REST calls | `admin-cli` |
 |`keycloak_force_install` | Remove pre-existing versions of service | `False` |
 |`keycloak_url` | URL for configuration rest calls | `http://{{ keycloak_quarkus_host }}:{{ keycloak_http_port }}` |
-|`keycloak_management_url` | URL for management console rest calls | `http://{{ keycloak_quarkus_host }}:{{ keycloak_management_http_port }}` |
```
Comment (guidograzioli) [inline roles/keycloak_quarkus/README.md:111]: If it is not used, please remove it from argument_specs as well

File: roles/keycloak_quarkus/meta/argument_specs.yml [hunk 3]
```diff
@@ -69,6 +69,18 @@ argument_specs:
                 default: false
                 description: "Ensure firewalld is running and configure keycloak ports"
                 type: "bool"
+            keycloak_service_restart_always:
+                default: false
+                description: "systemd restart always behavior of service"
```
Comment (guidograzioli) [inline roles/keycloak_quarkus/meta/argument_specs.yml:74]: Please add a note, if true it gets precedence over on-failure

**Review by guidograzioli — CHANGES_REQUESTED** [review]
Few minor changes please
And, while your are there, you're welcome to add yourself to https://github.com/ansible-middleware/keycloak/blob/main/galaxy.yml#L6

File: roles/keycloak_quarkus/README.md (hunk 2 above)
Comment (hwo-wd) [reply roles/keycloak_quarkus/README.md:111]: That's the case already ;-)

**Review by guidograzioli — APPROVED** [review]
LGTM

**Pull request merged**